processor/commonproc/preload_test.go

```diff
@@ -127,6 +127,65 @@ func TestExtractPreloadHeaders(t *testing.T) {
 				"Content-Type": []string{"text/html; charset=utf-8"},
 			},
 		},
+		{
+			name: "Multiple_MoreThanMaxPreloads",
+			url:  "https://example.com/hello.html",
+			resp: fmt.Sprint(
+				"HTTP/1.1 200 OK\r\n",
+				"Link: <https://example.com/photo1.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo2.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo3.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo4.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo5.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo6.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo7.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo8.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo9.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo10.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo11.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo12.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo13.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo14.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo15.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo16.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo17.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo18.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo19.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo20.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/photo21.jpg>;rel=\"preload\";as=\"image\"\r\n",
+				"Link: <https://example.com/>;rel=\"start\"\r\n",
```

Comment:
If you're looking for another fixit task after this PR, we should also strip this. :)

https://github.com/google/webpackager/blob/master/docs/cache_requirements.md#user-content-google-sxg-cache:~:text=All%20rel%20parameters%20must%20be%20either%20preload%20or%20allowed%2Dalt%2Dsxg.